JeffShepherd/fitness-tracker
# Chore/refactoring and cleanup

## What does this PR do?

- Refactors code where possible (mainly CSS)
- Removes white space from all files
- Add screenshots of app for use in ReadMe

## How should it be tested in the UI?

- Start app in browser with 'npm start' and ensure application still looks good for all screen sizes
- Check console for any errors
- Run tests with 'npm test' to ensure all tests still pass
